gameboard/game.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="game.css">
```

Comment:
For testing it is ok but normally, webpack should inject styles in your place. Besides, you decided to use SCSS so it would be better to import SCSS in the JS file related to the page.